File: modulo1/segmento3/apunte4/Ejercicio1.web/ejercicio1.js
function ejercicio1() {
    const number = parseInt(prompt('Ingrese un numero del 1 al 7 se devolvera un dia de semana:'));
    switch(number) {
    case 1:
    alert('Lunes');
    break;
    case 2:
    alert('Martes');
    break;
    case 3:
    alert('Miercoles');
    break;
    case 4:
    alert('Jueves');
    break;
    case 5:
    alert('Viernes');
    break;
    case 6:
    alert('Sabado');
    break;
    case 7:
    alert('Domingo');
    break;
    default:
    alert('ERROR: no es un dia de semana');
    break;
    }
    }
    
    

File: modulo1/segmento3/apunte4/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1 {
	public static void main(String args[]) {
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese un numero del 1 al 7 y se devolvera un dia de semana: ");
		Integer num = Integer.parseInt(keyboard.nextLine());

		switch(num){
			case 1:
				System.out.println("Lunes");
				break;

			case 2:
				System.out.println("Martes");
				break;
			case 3:
				System.out.println("Miercoles");
				break;
			case 4:
				System.out.println("Jueves");
				break;
			case 5:
				System.out.println("Viernes");
				break;
			case 6:
				System.out.println("Sabado");
				break;
			case 7:
				System.out.println("Domingo");
			default:
				System.out.print("ERROR: Numero ingresado incorrecto");
				break;
		}
	}
}
